src/blog/2022/009-es2018.md

```diff
@@ -0,0 +1,58 @@
+---
+
+title: Core Libraries Migrated To ES2018
+description: "Introducing bpmn-js@10, diagram-js@9, and dmn-js@13: The releases include migration of the core libraries to the modern syntax of ES2018."
+layout: blogpost.hbs
+slug: 2022-core-libraries-migrated-to-es2018
+author:
+- Maciej Barelkowski <https://github.com/barmac>
+published: 2022-09-14 12:00
+
+releases:
+  - 'bpmn-js@10.0.0'
+  - 'diagram-js@9.0.0'
+  - 'dmn-js@13.0.0'
+
+---
+
+<p class="introduction">
+  We are happy to announce the latest releases of our modeling toolkit which ship with modern JavaScript syntax, namely ES2018.
+</p>
+
+<!-- continue -->
+
+The releases introduce modern language constructs to the codebase, i.e. async/await, ES6-style classes, as well as rest/spread operator. We believe this will allow reduce the burden on contributors who won't need to limit to ES5 syntax anymore.
```

Comment:
I'd want us to clarify here that the _bundled_ output will contain ES2018 features. You could also say that the target runtime of our libraries is now ES2018 (which is every modern browser).

The main benefit for the user is smaller bundles (we don't shim/transpile back to ES5), while the main benefit on our end is maintainability and the ability to use these new language features freely.

Reply:
Please check now.